From 7a586c57c308107c3e63ae3cd199079de2bee8e6 Mon Sep 17 00:00:00 2001
From: Wolfgang Bauer <wbauer@tmo.at>
Date: Wed, 2 Jan 2019 10:54:42 +0100
Subject: [PATCH] Lower minimum cmake version

VERSION_GREATER_EQUAL is new in cmake 3.7.
So use NOT VERSION_LESS instead, which also works with older versions.
---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4cbbb09..a6301c3 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,6 +1,6 @@
 project(krename)
 
-cmake_minimum_required(VERSION 3.7)
+cmake_minimum_required(VERSION 3.0)
 # search packages used by KDE
 find_package(ECM 0.0.11 REQUIRED NO_MODULE)
 set(CMAKE_MODULE_PATH ${ECM_MODULE_PATH} ${ECM_KDE_MODULE_DIR})
@@ -61,7 +61,7 @@ set_package_properties(Taglib PROPERTIES
 find_package(LibExiv2 0.13) # AccessMode was added in 0.13
 set_package_properties(LibExiv2 PROPERTIES TYPE OPTIONAL)
 
-if (LibExiv2_VERSION VERSION_GREATER_EQUAL "0.27")
+if (NOT LibExiv2_VERSION VERSION_LESS "0.27")
     set(HAVE_LIBEXIV2_0_27 TRUE)
 endif()
 
-- 
2.16.4
